ServiceInnovationLab/pancake-backend
# Linking applications to properties, and making signing easier

closes #9 
# What has changed and why?
Cleaned up the display on the admin pages where we can seed application forms

If the application is for a property we know of (if it's in our database) then we display the address

We are continuing to accept valuation_ids that we don't recognise. We can stop that once the frontend is sending only known correct ones.

# How to test this change

set variables in a .env file, i suggest 
```
SIGNING_URL="http://127.0.0.1:4000/#/{token}"
COUNCIL_EMAIL="bob@example.com"
```

Then run the backend `bundle exec rails s`
and run the frontend `PORT=4000 npm start`

If you log in and look at rebate application forms you should see a button to bring up the singing interface. IT's a big green button

Also in this: you can no longer edit a rebate form. 

We have kept being able to delete them. We'll need to add that we can't delete after reconciling (or maybe we can, but it's an offline archiving process instead of destroying it).
